tests/test_callable.py

```diff
@@ -0,0 +1,141 @@
+import typing
+
+import attr
+import pytest
+
+from attrs_strict import AttributeTypeError, type_validator
+
+try:
+    from collections.abc import Callable
+except ImportError:
+    from collections import Callable
+
+
+@pytest.fixture()
+def simple_callable_type():
+    class UnAnnotatedCallable(Callable):
+        def __call__(self):
+            pass
+
+    return UnAnnotatedCallable
+
+
+@pytest.fixture
+def annotated_callable_int_str_type():
+    class AnnonatedCallable(Callable):
+        def __call__(self, a, b):
+            return str(a + b)
+
+        __call__.__annotations__ = {"a": int, "b": int, "return": str}
```

Comment:
why not specify the annotation instead of modifying `__annotations__`?

Reply:
For `python2` compatibility